.gitlab-ci.yml

```diff
@@ -101,12 +101,14 @@ Review:
   before_script:
     - kubectl config use-context ${AGENT}
     # Check for review namespace and remove the default backend ingress
-    - kubectl get namespace ${NS} || (echo "Please create '${NS}' branch in ensembl-client and restart this pipeline" && exit 1)
+    - kubectl get namespace ${NS} > /dev/null || (echo "Please create '${NS}' branch in ensembl-client and restart this pipeline" && exit 1)
```

Comment:
Yes, (at least for Tools API) the assumption is that if the namespace is not there it means we have forgotten to create the client-side review app, and that's almost always the case:
* Tools API has no endpoint that is meant to be used independently from the client
* If Tools API would create the review namespace here, client-side deployment would fail (because it will try to create ns, hence the check)
* We could add another job if we want ToolsAPI-only review app, but why? And how do we control which kind of review gets deployed?
* If we really want to create Tools API-only review app, we can still do it by creating namespace manually (a single command) and then deploying the review branch.
Same approach in Genome Browser (other backends don't have review deployment yet).

Reply:
I get what you are saying; it just seems that there is something wrong with our review deployment setup that we need to switch to a different repo and create a new branch in order to create a review deployment for code in this repo. I feel like there should be some kind of button, ideally with a bunch of associated selectors, that would deploy the desired combination of branches of different services. I am surprised that there isn't a clean solution for this.

_(Not a blocker for the PR, of course)_